wiki/Tournaments/oPHN/2022_mania/en.md

```diff
@@ -0,0 +1,789 @@
+---
+tags:
+  - PHNM2022
+  - PHNM 2022
+---
+
+# osu! Philippines Nationals 2022 - osu!mania 4K/7K
+
+![PHNM2022 banner](img/banner.png)
+
+The **osu! Philippines Nationals 2022 - osu!mania 4K/7K** (***PHNM 2022***) was a one-on-one, osu!mania 4K and 7K tournament made for osu! players based in the Philippines or of Filipino heritage.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-04-17/2022-05-08 |
+| Qualifiers | 2022-05-22 |
+| Group Stage Week 1 | 2022-06-04/2022-06-05 |
+| Group Stage Week 2 | 2022-06-11/2022-06-12 |
+| Round of 16 | 2022-06-18/2022-06-19 |
+| Quarterfinals | 2022-06-25/2022-06-26 |
+| Semifinals | 2022-07-02/2022-07-03 |
+| Finals week 1 | 2022-07-09/2022-07-10 |
+| Finals week 2 | 2022-07-16/2022-07-17 |
```

Comment:
```suggestion
| Finals | 2022-07-09/2022-07-10 |
| Grand Finals | 2022-07-16/2022-07-17 |
```
The page mentions both 'Grand Finals' and 'Finals week 2'. This, along with the table in the rules section, are the only places where 'Finals week 2' is mentioned. Given that the sheet also calls it Grand Finals, I think we can go with that.
```